.github/workflows/release.yml

```diff
@@ -203,3 +203,41 @@ jobs:
           for f in $(find dist/ -name '*.tgz'); do
             gh release upload $tag $f
           done
+
+      - name: Add charts to branch
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          version=$(jq -r '.version' <<< "${{ steps.goreleaser.outputs.metadata }}")
+          branch_version=$(cut -d'.' -f1,2 <<< "$version")
+          charts_branch=charts/$branch_version
+
+          if [ ! -e ~/.gitconfig ]; then
+            git config --global user.name "fleet-bot"
+            git config --global user.email fleet@suse.de
+          fi
+
+          echo "publishing helm chart in the branch $charts_branch"
+          if ! git show-ref --quiet "refs/heads/$charts_branch"; then
+            git checkout --orphan "$charts_branch"
+            git rm -rf .
+
+            echo "# Fleet Helm Charts for $branch_version versions" > README.md
+            echo "The documentation is centralized in a unique place, checkout https://fleet.rancher.io/." >> README.md
+
+            git checkout main -- .gitignore
+
+            git add README.md .gitignore
+            git commit -m "Initial commit for $charts_branch"
```

Comment:
This initial commit will remove all contents from the index.
Without being knowledgeable enough in git internals, I wonder how repeating this process for multiple release branches, in addition to adding charts, might increase this repository's `.git` directory :thinking:

Reply:
Yes, what I meant to say was not so much that we'd need to keep all those unneeded files around, but rather that using a different repository for this could make sense, similarly to what we have with [`rancher/fleet-helm-charts`](https://github.com/rancher/fleet-helm-charts/).